models/events/webhook/github/GithubPush.js

```diff
@@ -0,0 +1,5 @@
+module.exports = require('../../../Model').extend({
+  properties: {
+    compare: String
```

Comment:
`compare` is a real property that a github push event sends you. I added this as a placeholder and I think it's fine to leave it for now. I wanted a way to add real integration tests.